requirements.txt

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
@woodt where is this file being used?
Perhaps add an explicit install of these versions in the Dockerfile?

Reply:
Hi ... for the moment, this is only used by the vulnerability scanner.

Reply:
Wouldn't it make sense to use in the image exactly the same package versions that we've tested for vulnerabilities?
Otherwise we might use a newer version that has a vulnerability while testing for the old, safe version.

Reply:
Absolutely ... I was worried about catching every time dependencies were installed.  (Also, I never use conda, so I was a little scared :-)

My best guess would be to add *pip install*'s of the specific requirements to all the Dockerfiles.  Do you think that would catch all the installations?